Tests/SmokeDynamoDBTests/DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests.swift

```diff
@@ -585,17 +539,17 @@ class DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests: XCTestC
             XCTFail("Unexpected error thrown: \(error).")
         }
         
-        let secondRetrievedItem: TestTypeADatabaseItem = try! table.getItem(forKey: key).wait()!
+        let secondRetrievedItem: TestTypeADatabaseItem = try await table.getItem(forKey: key)!
         
         XCTAssertEqual("sortId", secondRetrievedItem.compositePrimaryKey.sortKey)
         // Check the item hasn't been updated
         XCTAssertEqual("firstly", secondRetrievedItem.rowValue.firstly)
         XCTAssertEqual("secondly", secondRetrievedItem.rowValue.secondly)
     }
     
-    func testUpdateItemConditionallyAtKeyWithFailingUpdateWithAsyncItemProvider() {
-        let wrappedTable = InMemoryDynamoDBCompositePrimaryKeyTable(eventLoop: eventLoop)
-        let table = SimulateConcurrencyDynamoDBCompositePrimaryKeyTable(wrappedDynamoDBTable: wrappedTable, eventLoop: eventLoop,
+    func testUpdateItemConditionallyAtKeyWithFailingUpdateWithAsyncItemProvider() async throws {
```

Comment:
This test is now virtually identical to `testUpdateItemConditionallyAtKeyWithFailingUpdateWithItemProvider`.

Reply:
Removed